gadget/mountedfilesystem.go

```diff
@@ -345,12 +345,21 @@ func maybePreserveManagedBootAssets(mountPoint string, ps *LaidOutStructure) ([]
 	if ps.Role != SystemSeed && ps.Role != SystemBoot {
 		return nil, nil
 	}
+	// TODO:UC20: this code should no longer be needed when we use
```

Comment:
but also yes I agree with this TODO, regardless of the state of unit tests around this code